ui/src/app/pools/views/Pools.js

```diff
@@ -69,7 +69,7 @@ const generateRows = (
                   "Cannot delete a pool that contains machines.")
               }
               editDisabled={!row.permissions.includes("edit")}
-              editPath={poolsURLs.edit({ edit: row.id })}
+              editPath={poolsURLs.edit({ id: row.id })}
```

Comment:
This is just a driveby fix for going to the correct url.

Reply:
whoops, how did that happen? :grimacing:

Reply:
This is what happens when it's not a TypeScript file!